test/e2e/scenarios/node-labels.go

```diff
@@ -82,7 +82,7 @@ func labeledDeployTestSuite() {
 		// wait till operator reconcile csi
 		// operator has to receive NodeUpdate request and label new nodes to expand node-daemonset
 		// if new pods aren't created in time, WaitForPodsRunningReady will be skipped
-		deadline := time.Now().Add(time.Second * 30)
+		deadline := time.Now().Add(2 * time.Minute)
```

Comment:
Why time increased from 30 seconds up to 2 minutes?

Reply:
We have some fails in CI when CSI Operator has no time to label nodes. If it works correctly, the loop will be stopped after success, if no, we will have more info about a failure